services/cinema_hall.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: str,
+        hall_seats_in_row: int
+) -> QuerySet:

```

Comment:
maybe `-> CinemaHall` would be better type for the function?

Reply:
done